soc/arm/st_stm32/common/Kconfig.defconfig.series

```diff
@@ -17,6 +17,12 @@ config SYS_CLOCK_TICKS_PER_SEC
 	default 4096 if STM32_LPTIM_TIMER && STM32_LPTIM_CLOCK_LSE
 	default 4000 if STM32_LPTIM_TIMER && STM32_LPTIM_CLOCK_LSI
 
+# Disable null pointer exception detection by default as long as debug
+# mode is left active by west runners (openocd mainly)
+choice CORTEX_M_DEBUG_NULL_POINTER_EXCEPTION_DETECTION
```

Comment:
ideally we should link a ticket here for openOCD

Reply:
Right, fixed